### PR TITLE
ID localization for debug application cluster - resource usage monitoring

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR translates `/docs/tasks/debug-application-cluster/resource-usage-monitoring/` as mentioned in #22296

### CODE REVIEW

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+karakteristik klaster secara keseluruhan. Kubernetes memberikan detail
+informasi tentang penggunaan sumber daya dari aplikasi pada setiap level ini.
+Informasi ini memungkinkan kamu untuk mengevaluasi kinerja aplikasi kamu dan
+mengevaluasi di mana kemacetan dapat dihilangkan untuk meningkatkan kinerja secara keseluruhan.
+
+
+
+<!-- body -->
+
+Di Kubernetes, pemantauan aplikasi tidak bergantung pada satu solusi pemantauan saja. Pada klaster baru, kamu bisa menggunakan _pipeline_ [metrik sumber daya](#resource-metrics-pipeline) atau _pipeline_ [metrik penuh](#full-metrics-pipeline) untuk mengumpulkan statistik pemantauan.
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:21]: Yes, @sftim. Thank you for your comment.

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+content_type: concept
+title: Perangkat untuk Memantau Sumber Daya
+---
+
+<!-- overview -->
+
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:8]: nit: verbs needed
```suggestion
Untuk melakukan penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+memahami bagaimana aplikasi berperilaku ketika aplikasi tersebut digelar. Kamu bisa memeriksa
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:9]: digelar doesn't seem familiar, how about adding parentheses (_deploy_) or change to "dijalankan"?

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+kinerja aplikasi dalam klaster Kubernetes dengan memeriksa Container,
+[Pod](/id/docs/user-guide/pods), [Service](/id/docs/user-guide/services), dan
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:11]: we don't have these yet
```suggestion
[Pod](/docs/user-guide/pods), [Service](/docs/user-guide/services), dan
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (girikuncoro) [reply content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:21]: @ariscahyadi you haven't addressed this yet
```suggestion
Di Kubernetes, pemantauan aplikasi tidak bergantung pada satu solusi pemantauan saja. Pada klaster baru, kamu bisa menggunakan _pipeline_ [metrik sumber daya](#pipeline-metrik-sumber-dayae) atau _pipeline_ [metrik penuh](#pipeline-metrik-penuh) untuk mengumpulkan statistik pemantauan.
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+
+<!-- body -->
+
+Di Kubernetes, pemantauan aplikasi tidak bergantung pada satu solusi pemantauan saja. Pada klaster baru, kamu bisa menggunakan _pipeline_ [metrik sumber daya](#resource-metrics-pipeline) atau _pipeline_ [metrik penuh](#full-metrics-pipeline) untuk mengumpulkan statistik pemantauan.
+
+## _Pipeline_ Metrik Sumber Daya
+
+_Pipeline_ metrik sumber daya menyediakan sekumpulan metrik terbatas yang terkait dengan
+komponen-komponen klaster seperti _controller_ [Penyekala Pod Horisontal](/id/docs/tasks/run-application/horizontal-pod-autoscale), begitu juga dengan utilitas `kubectl top`.
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:26]: HPA is kubernetes object
and typo on link
```suggestion
komponen-komponen klaster seperti _controller_ [HorizontalPodAutoscaler](/id/docs/tasks/run-application/horizontal-pod-autoscaler), begitu juga dengan utilitas `kubectl top`.
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+Node, mengelola Pod dan Container yang berjalan pada sebuah mesin. Kubelet
+menerjemahkan setiap Pod ke Container yang menyusunnya dan mengambil masing-masing
+statistik penggunaan untuk setiap Container dari _runtime_ Container melalui
+antarmuka _runtime_ Container. Kubelet mengambil informasi ini dari cAdvisor yang terintegrasi
+untuk pengintegrasian Docker yang lama. Hal ini yang kemudian memperlihatkan
+statistik penggunaan sumber daya dari kumpulan Pod melalui API sumber daya _metrics-server_.
+API ini disediakan pada `/metrics/resource/v1beta1` pada kubelet yang terautentikasi dan
+porta _read-only_.
+
+## _Pipeline_ Metric Penuh
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:44]:
```suggestion
## _Pipeline_ Metrik Penuh
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+
+_Pipeline_ metrik penuh memberi kamu akses ke metrik yang lebih banyak. Kubernetes bisa
+menanggapi metrik ini secara otomatis dengan mengubah skala atau mengadaptasi klaster
+berdasarkan kondisi saat ini, dengan menggunakan mekanisme seperti penyekala Pod horisontal.
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:48]:
```suggestion
berdasarkan kondisi saat ini, dengan menggunakan mekanisme seperti HorizontalPodAutoscaler.
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+_Pipeline_ pemantauan mengambil metrik dari kubelet dan
+kemudian memgekspos ke Kubernetes melalui adaptor dengan mengimplementasikan salah satu dari API
+`custom.metrics.k8s.io` atau API `external.metrics.k8s.io`.
+ 
+
+[Prometheus](https://prometheus.io), sebuah proyek CNCF, yang dapat secara asli memonitor Kubernetes, Node, dan Prometheus itu sendiri.
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:54]: what does "secara asli" mean?

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+Proyek _Pipeline_ metrik penuh yang bukan merupakan bagian dari CNCF berada di luar ruang lingkup dari dokumentasi Kubernetes.
```
Comment (girikuncoro) [inline content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:55]: nit
```suggestion
Proyek _pipeline_ metrik penuh yang bukan merupakan bagian dari CNCF berada di luar ruang lingkup dari dokumentasi Kubernetes.
```

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:11]: Ok. But, I think we need to create this script `update-user-guide-links.py` to handle redirection "user-guide"

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md (hunk 5 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:26]: Ok!

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md (hunk 3 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:9]: Ok!

File: content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md (hunk 8 above)
Comment (ariscahyadi) [reply content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:54]: "secara asli" mean "native" :)

**Comment by ariscahyadi** [comment]
> Thanks for localizing this page @ariscahyadi . I've left few comments and found some of links to ID pages that we don't have yet. The rest is nit.

Hi @girikuncoro ... I have already addressed all the comments. Please check!

**Comment by girikuncoro** [comment]
Thanks @ariscahyadi !

/lgtm
/approve

**Pull request merged**